### DISCUSSION TITLE
CrowdSec Script Error

### DISCUSSION THREAD

**Discussion by ghost** [tteck/Proxmox#2047] [Help Needed] [answered]
I am trying to install CrowdSec. Error: "⚠️ Can't Install on Proxmox" What am I doing wrong ?

**Comment by tteck** [comment, marked as answer]
To Install CrowdSec, ⚠️ run the command below in the **LXC console**.

**Reply by ghost** [reply]
Oh IN a Container? With what requirements?

**Reply by tteck** [reply]
Any existing LXC
